### ISSUE TITLE
[Accessibility] Material tabs have almost no visible keyboard focus sate

### ISSUE THREAD

**Issue by 6TELOIV** [react-navigation/react-navigation#12594]
### Current behavior

When tabbing through the vertical tabs buttons, they have virtually no visible indication of focus.

The Profile tab is foused here
![three tabs in light theme: home active, profile focused, settings unfocused. Profile has slight 1px borders in the corners visible](https://github.com/user-attachments/assets/ca3bfa6c-33bf-4b12-a7c0-2d9a9955a494)

The issue is particularly bad with the dark theme (which is shown in the reproduction)

The Profile tab is foused here
![three tabs in dark theme: home active, profile focused, settings unfocused. Profile has no visible focus indication](https://github.com/user-attachments/assets/2f5ddcd4-4df1-4a6e-bf5b-01c0f63f2843)

### Expected behavior

The interactive elements should have a clear and distinct focus state. At minimum, this to me means a focus border. Alternatively, you could show the hover darkening/lightening on keyboard focus, though sticking to browser default is usually best.

I was able to get this appearance just by turning off `overflow: hidden` on the tab bar item:

![three tabs in dark theme: home active, profile focused, settings unfocused. Profile has the default browser focus ring visible](https://github.com/user-attachments/assets/fa5afea9-61e1-407e-9504-143b1a3b2ffa)

### Reproduction

https://snack.expo.dev/@6teloiv/bottom-tabs-material---keyboard-focus-issue

### Platform

- [ ] Android
- [ ] iOS
- [x] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [x] @react-navigation/bottom-tabs
- [ ] @react-navigation/drawer
- [ ] @react-navigation/material-top-tabs
- [ ] @react-navigation/stack
- [ ] @react-navigation/native-stack
- [ ] react-native-drawer-layout
- [ ] react-native-tab-view

### Environment

- [x] I've removed the packages that I don't use

| package                                | version |
| -------------------------------------- | ------- |
| react-native-screens | ~4.4.0 |
| @react-navigation/native | 7.1.9 |
| @react-navigation/bottom-tabs | 7.3.13 |
| react-native-safe-area-context | 4.12.0 |

**Comment by UtkarshAnandd** [comment]
hey is it open to work on or closed ?
